Sources/ScipioKit/Runner.swift

```diff
@@ -154,6 +161,34 @@ extension Runner {
                 self.enableLibraryEvolution = enableLibraryEvolution
                 self.frameworkModuleMapGenerationPolicy = frameworkModuleMapGenerationPolicy
             }
+
+            #if compiler(>=6.0)
+            public init(
+                buildConfiguration: BuildConfiguration = .release,
+                platforms: PlatformSpecifier = .manifest,
+                isSimulatorSupported: Bool = false,
+                isDebugSymbolsEmbedded: Bool = false,
+                frameworkType: FrameworkType = .dynamic,
+                extraFlags: ExtraFlags? = nil,
+                extraBuildParameters: [String: String]? = nil,
+                enableLibraryEvolution: Bool = false,
+                frameworkModuleMapGenerationPolicy: FrameworkModuleMapGenerationPolicy = .autoGenerated,
+                environment: Environment? = nil
+            ) {
```

Comment:
Alternatively, instead of passing the `Environment` type, we could pass a plain `[String: String]` dictionary containing the environment variables. This dictionary could later be mapped to the Environment type, allowing us to avoid the checks.

However, once support for `Xcode 15` is dropped, these checks will be removed anyway. Just tell me what you prefer.

Reply:
> However, once support for Xcode 15 is dropped, these checks will be removed anyway. 

Does this mean SwiftPM for Swift 5.x has a different interface?

I prefer to use a plane `[String: String]` in this case. Because `Runner` will be published to the external packages so we wish to keep fewer dependencies in the public interfaces as possible.
Using `Environment` here, we need to depend on SwiftPM from the caller side.

There is a problem with the keys don't become type-safe, but we may add a runtime check.

Reply:
That's a good point, then I'll switch over to [String:String].

One thing, I'm on vacation till the end of the next week. Once back, I'll make the changes!

Reply:
Never mind. Enjoy!

Reply:
Thanks! I'll answer to the other comments when I'm back.

Reply:
Hey, I'm finally back, and I've made the changes we discussed above.